content/reference/plugins/grafana/index.md

```diff
@@ -175,7 +175,7 @@ The returned percentiles are: min, p25, p50, p75, p80, p85, p90, p95, p99, p999,
 
 ### Part 2
 
-| Metrics name                   | remote             | hostname           | injector           | scope              |
+| Metrics name                   | remote             | hostname           | load generator     | scope              |
```

Comment:
documentation.png shows an error where Documentation overlaps with the link icon. I don't have it on my computer.